server.go

```diff
@@ -23,6 +25,17 @@ func main() {
 		mongoHost = "localhost"
 	}
 
+	codeLookupFlag := flag.Bool("loadCodes", false, "flag to enable download of icd-9 code lookup")
+	codeLookupURL := flag.String("lookupURL", "https://www.cms.gov/Medicare/Coding/ICD9ProviderDiagnosticCodes/Downloads/ICD-9-CM-v32-master-descriptions.zip", "url for icd-9 code definition zip")
```

Comment:
I suggest we name this "icd9URL" -- because when we add support for ICD-10 (and others), then we'll need multiple flags to specify the locations.